tracer/src/Datadog.Trace/Activity/Handlers/DefaultActivityHandler.cs

```diff
@@ -43,22 +43,28 @@ public void ActivityStarted<T>(string sourceName, T activity)
             ulong? spanId = null;
             string? rawTraceId = null;
             string? rawSpanId = null;
+            bool isRemoteContext = false;
```

Comment:
I would have used `isSeparatedContext` because remote means something else in my mind. But I'm not an English speaker and I'm the worst person for naming things (besides my kids :p), so don't trust me on this :)